mavisp/modules.py

```diff
@@ -768,6 +767,109 @@ def ingest(self, mutations):
         if len(warnings) > 0:
             raise MAVISpMultipleError(warning=warnings, critical=[])
 
+class TaccDenovoPhospho(DenovoPhospho):
+    
+    expected_files = ['aggregated_filtered_output.csv', 'acc_REL.csv']
+    
+    def __init__(self, data_dir=None, module_dir="denovo_phospho"):
+        super().__init__(data_dir=data_dir,module_dir=module_dir)
+        self.sasa_fname = 'acc_REL.csv'
+
+    def _parse_sas(self, fname):
+        
+        try:
+            sas_data = pd.read_csv(fname, usecols=['residue', 'acc_average'])
+        except Exception as e:
+            raise Exception(f"Failed to load and parse '{fname}': {e}")
+
+        sas_data.set_index('residue', inplace=True)
+        sas_data.rename(columns={'acc_average': 'sas_sc_rel'}, inplace=True)
+
+        return sas_data.fillna(pd.NA)
```

Comment:
do we need to `fillna` for any specific reason? Do we have nans/NAs in our input file?

Reply:
we don't need, should I remove it? I'ts there because I copy the structure from PTM module and there we have this fillna